dgraph/cmd/alpha/run.go

```diff
@@ -149,6 +145,16 @@ they form a Raft group and provide synchronous replication.
 				"worker in a failed state. Use -1 to retry infinitely.").
 		String())
 
+	// Cache flags.
+	flag.String("cache", worker.CacheDefaults, z.NewSuperFlagHelp(worker.CacheDefaults).
+		Head("Cache options").
+		Flag("size-mb",
+			"Total size of cache (in MB) to be used in Dgraph.").
+		Flag("percentage",
+			"Cache percentages summing up to 100 for various caches (FORMAT: PostingListCache,"+
+				"PstoreBlockCache,PstoreIndexCache,WAL)").
```

Comment:
This needs to be changed. The pstoreblock cache and pstore index cache values are being fetched from the badger super flag. We don't need them over here.

Also, the wal is no longer supported.

Reply:
Removed the wal cache from flag. Also, we decided to keep the cache as it is.